docs/source/community/mission-scope.md

```diff
@@ -3,7 +3,7 @@
 
 ## Mission
 
-[movement](movement-website:) aims to **facilitate the study of animal behaviour in neuroscience** by providing a suite of **Python tools to analyse body movements** across space and time.
+[movement](../index.md) aims to **facilitate the study of animal behaviour in neuroscience** by providing a suite of **Python tools to analyse body movements** across space and time.
```

Comment:
While we are at it, any chance we can replace this type of relative links with [explicit targets](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html)? Should be more robust to files moving around.

Reply:
Resolved in f890628